src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
```

Comment:
Does the `ext.flutter.widgetInspector` toggle anything on device, or is it a one-shot thing to request an on-device inspector? (/cc @jacob314)